packages/vital_core/lib/services/link_service.dart

```diff
@@ -65,6 +65,16 @@ abstract class LinkService extends ChopperService {
     @Header('LinkToken') required String linkToken,
   });
 
+  @Get(
+      path: 'link/connect/{provider}',
+      headers: {"X-Vital-SDK-No-Redirect": "1"})
```

Comment:
When the header is present, backend will simply 200 OK instead of redirecting to the LinkToken's redirect URL.